### PR TITLE
feat: mobile hover bag

### PR DESCRIPTION
**Pull request by JackShort**
* implemented the hover bag for mobile

### CODE REVIEW

File: src/nft/components/bag/BagRow.tsx [hunk 1]
```diff
@@ -64,6 +65,20 @@ export const BagRow = ({ asset, usdPrice, removeAsset, showRemove, grayscale, is
...
+          <Box
+            display={showRemove ? 'block' : 'none'}
+            className={styles.removeAssetOverlay}
+            onClick={(e: MouseEvent) => {
+              e.preventDefault()
+              e.stopPropagation()
+              removeAsset(asset)
+            }}
+            transition="250"
+            style={{ opacity: isMobile ? '1' : '0' }}
```
Comment (cbachmeier) [inline src/nft/components/bag/BagRow.tsx:77]: Opacity can be sprinkles, also we could add the mobile check to the display prop instead

File: src/nft/components/bag/BagRow.tsx [hunk 2]
```diff
@@ -91,7 +106,7 @@ export const BagRow = ({ asset, usdPrice, removeAsset, showRemove, grayscale, is
             {asset.collectionIsVerified && <VerifiedIcon className={styles.icon} />}
           </Row>
         </Column>
-        {cardHovered && showRemove && (
+        {cardHovered && showRemove && !isMobile && (
```
Comment (cbachmeier) [inline src/nft/components/bag/BagRow.tsx:109]: nit: Could make this condition a const like `showRemoveButton` and then the check for showing the price could just be `!showRemoveButton`

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/nft/components/bag/MobileHoverBag.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+import * as styles from './MobileHoverBag.css'
+
+export const MobileHoverBag = () => {
+  const itemsInBag = useBag((state) => state.itemsInBag)
+  const toggleBag = useBag((state) => state.toggleBag)
+
+  const { pathname } = useLocation()
+  const isNFTSellPage = pathname.startsWith('/nfts/sell')
+  const isNFTPage = pathname.startsWith('/nfts')
```
[the remark this answers is not in the document]
Comment (JackShort) [reply src/nft/components/bag/MobileHoverBag.tsx:21]: ah yes forgot to remove this when moving it from the `topLevelModals`

File: src/nft/components/bag/MobileHoverBag.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                item.asset.updatedPriceInfo ? item.asset.updatedPriceInfo.ETHPrice : item.asset.priceInfo.ETHPrice
+              )
+            )
+          : total,
+      BigNumber.from(0)
+    )
+    const totalUsdPrice = usdPrice ? parseFloat(formatEther(totalEthPrice)) * usdPrice : undefined
+
+    return { totalEthPrice, totalUsdPrice }
+  }, [itemsInBag, usdPrice])
```
[the remark this answers is not in the document]
Comment (JackShort) [reply src/nft/components/bag/MobileHoverBag.tsx:41]: good idea

File: src/nft/components/bag/MobileHoverBag.tsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+          <Row gap="8">
+            <Box className={body}>{`${formatWeiToDecimal(totalEthPrice.toString())}`}</Box>
+            <Box color="textSecondary" className={bodySmall}>{`${ethNumberStandardFormatter(
+              totalUsdPrice,
+              true
+            )}`}</Box>
+          </Row>
+        </Column>
+      </Row>
+      <Box className={styles.viewBagButton} onClick={toggleBag}>
```
[the remark this answers is not in the document]
Comment (JackShort) [reply src/nft/components/bag/MobileHoverBag.tsx:85]: I think so but its not in the design

**Review by cbachmeier — APPROVED** [review]
LGTM

**Pull request merged**